### PR TITLE
User login page

### PR DESCRIPTION
**Pull request by gourabatsap**

### CODE REVIEW

File: App/app/android/build.gradle [hunk 1]
```diff
@@ -26,6 +26,6 @@ subprojects {
     project.evaluationDependsOn(':app')
 }
 
-task clean(type: Delete) {
+tasks.register("clean", Delete) {
```
Comment (abhianuj) [inline App/app/android/build.gradle:29]: why is this needed??

File: App/app/android/app/src/debug/AndroidManifest.xml [hunk 2]
```diff
@@ -4,5 +4,6 @@
          the Flutter tool needs it to communicate with the running application
          to allow setting breakpoints, to provide hot reload, etc.
     -->
-    <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
 </manifest>
```
Comment (abhianuj) [inline App/app/android/app/src/debug/AndroidManifest.xml:9]: Please remove these changes, as it is not required to access the network state.

File: App/app/android/app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -1,5 +1,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.example.app">
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```
Comment (abhianuj) [inline App/app/android/app/src/main/AndroidManifest.xml:4]: Same as above
Please remove these changes, as it is not required to access the network state.

File: App/app/pubspec.yaml [hunk 4]
```diff
@@ -31,7 +31,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-
+  get_storage: ^2.1.1
```
Comment (abhianuj) [inline App/app/pubspec.yaml:34]: We are using secure storage so this is not needed, I will take care of this part.

File: App/app/lib/main.dart [hunk 5]
```diff
@@ -1,10 +1,16 @@
+import 'dart:io';
+
```
Comment (abhianuj) [inline App/app/lib/main.dart:2]: Remove all the changes from main.dart.
As i have allready implemnetd a multiprovider in my local and we will be using that.

Thanks :D

**Comment by abhianuj** [comment]
Also please remove the unused leaf pic, there are two and only one is used.

**Review by abhianuj — CHANGES_REQUESTED** [review]

File: App/app/android/build.gradle (hunk 1 above)
Comment (gourabatsap) [reply App/app/android/build.gradle:29]: These all are pre generated files from android studio ideally they shouldn't have been pushed and gitignored from beginning.

File: App/app/android/app/src/debug/AndroidManifest.xml (hunk 2 above)
Comment (gourabatsap) [reply App/app/android/app/src/debug/AndroidManifest.xml:9]: Won't do

File: App/app/pubspec.yaml (hunk 4 above)
Comment (gourabatsap) [reply App/app/pubspec.yaml:34]: Won't do

File: App/app/pubspec.lock [hunk 6]
```diff
@@ -5,51 +5,74 @@ packages:
     dependency: transitive
```
[the remark this answers is not in the document]
Comment (gourabatsap) [reply App/app/pubspec.lock:5]: Won't do

File: App/app/lib/main.dart (hunk 5 above)
Comment (gourabatsap) [reply App/app/lib/main.dart:2]: Won't do

**Comment by gourabatsap** [comment]
Since your local changes were not pushed I can't anticipate your local changes so please get it into main and do the needful in your local 🤣

File: App/app/android/build.gradle (hunk 1 above)
Comment (abhianuj) [reply App/app/android/build.gradle:29]: Please remove it :D

File: App/app/android/app/src/main/AndroidManifest.xml (hunk 3 above)
Comment (abhianuj) [reply App/app/android/app/src/main/AndroidManifest.xml:4]: do it

File: App/app/pubspec.yaml (hunk 4 above)
Comment (abhianuj) [reply App/app/pubspec.yaml:34]: Abey do it

File: App/app/pubspec.lock (hunk 6 above)
[the remark this answers is not in the document]
Comment (abhianuj) [reply App/app/pubspec.lock:5]: Do it

File: App/app/lib/main.dart (hunk 5 above)
Comment (abhianuj) [reply App/app/lib/main.dart:2]: Do it

**Comment by abhianuj** [comment]
> Since your local changes were not pushed I can't anticipate your local changes so please get it into main and do the needful in your local 🤣

That is not the point,
We need to follow a different approach for this,
That's why i am asking 😞

**Comment by abhianuj** [comment]
Good work :D
